tests/functional_small/solvers_nonlinear/qr/fixtures.hpp

```diff
@@ -7,8 +7,11 @@
 #include "qr_r9c4_gold.hpp"
 
 #ifdef PRESSIO_ENABLE_TPL_TRILINOS
+#ifdef PRESSIO_ENABLE_EPETRA
+#include "Epetra_MpiComm.h"
+#endif // PRESSIO_ENABLE_EPETRA
 #include "Eigen/Dense"
-#endif
+#endif // PRESSIO_ENABLE_TPL_TRILINOS
```

Comment:
Should this `Eigen` include be wrapped in `PRESSIO_ENABLE_TPL_TRILINOS`?

Reply:
can you please move it outside of the if

Reply:
Done